README.md

```diff
@@ -20,9 +20,9 @@ Wagtail-Flags adds a Wagtail admin UI and Wagtail Site-based condition on top of
 
 ## Dependencies
 
-- Python 2.7+, 3.6+
-- Django 1.11+, 2.0+
-- Wagtail 1.13+, 2.0+
+- Python 3.6, 3.8
+- Django 1.11, 2.0, 2.2
+- Wagtail 1.13, 2.3, 2.8
```

Comment:
I think we probably want to leave these as ranges for the 2.x versions of Django and Wagtail. We may not be running automated tests against each version in the range, but we still support them.